app/exporthtmldialog.cpp

```diff
@@ -32,8 +33,9 @@ ExportHtmlDialog::ExportHtmlDialog(const QString &fileName, QWidget *parent) :
 
     if (!fileName.isEmpty()) {
         QFileInfo info(fileName);
-        QString exportFileName = info.absoluteFilePath().replace(info.suffix(), "html");
-        ui->exportToLineEdit->setText(exportFileName);
+        QString absFilePath = info.absoluteFilePath();
+        QString exportFileName = absFilePath.mid(0, absFilePath.lastIndexOf('.')) + ".html";
```

Comment:
Does this also work if there is no extension in the file name? Maybe we should use `QFileInfo::baseName()` instead, which probably also makes the code more readable. Something like `QString exportFileName = info.absolutePath() + info.baseName() + ".html";` (Untested!)